app/fastlane/Fastfile

```
@@ -53,61 +58,121 @@ platform :ios do
+    if test_mode
+      UI.important("🧪 TEST MODE: Skipping TestFlight upload")
+      UI.success("✅ Build completed successfully!")
+      UI.message("📦 IPA path: #{result[:ipa_path]}")
+    else
+      upload_to_testflight(
+        api_key: result[:api_key],
+        distribute_external: true,
+        # Only external TestFlight groups are valid here
+        groups: ENV["IOS_TESTFLIGHT_GROUPS"].split(","),
+        changelog: "",
+        skip_waiting_for_build_processing: false,
```

RuboCop (1.76.1)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)